web/.gitignore

```diff
@@ -4,8 +4,6 @@ dist-ssr
 
 _build/pages.ts
 
-src/contracts.json
-
```

Comment:
did you add this? i'm not sure we need it in here but if you think we do, let me know

Reply:
Think people just have to compile their own contracts.json so it can be left out